### DISCUSSION TITLE
Why can't I disable display a window when opening a magnet link?

### DISCUSSION THREAD

**Discussion by hollandgh** [transmission/transmission#5445] [Q&A] [answered]
In settings → Transfers → Add window I can disable "Display a window when opening a torrent file". But the option for that for magnet links is permanently grayed out. Why? It's annoying to have to click through the same old windows every time I download a magnet link. I'll probably switch to another client just because of this.

**Comment by nevack** [comment, marked as answer]
https://github.com/transmission/transmission/issues/5034#issuecomment-1444366215
